flow/templates/print_status.jinja

```diff
@@ -0,0 +1,64 @@
+{# The following variables are available to all scripts. #}
+{% if overview %}
+{% set format_progress_bar = ("%%-%ss  %%-%ss" | format(column_width_label,column_width_bar,)) %}
```

Comment:
Minor change - `flake8` requires _spaces after commas_ in the Python code, and I think it would be good to style this template's logic in the same way. Also I would drop the last comma if it's not required.